src/components/Icon/Expensicons.js

```diff
@@ -13,6 +13,7 @@ import Clipboard from '../../../assets/images/clipboard.svg';
 import Close from '../../../assets/images/close.svg';
 import CreditCard from '../../../assets/images/creditcard.svg';
 import DownArrow from '../../../assets/images/down.svg';
+import DownArrowSmall from '../../../assets/images/down-small.svg';
```

Comment:
I don't think you actually need a new asset for this. Why not just use the existing down arrow and just resize it to 12x12? That's what we have in the Figma file.

Reply:
Good point. I have removed the new asset and now reusing the old one in the latest commit.

Thanks.